### DISCUSSION TITLE
(erlay tx reconciliation sets code review part - 2) June 13 2024

### DISCUSSION THREAD

**Discussion by VirdiJatin** [Bitshala/BitcoinCore-PR-Review-Club#116] [PR Reviews]

**Comment by VirdiJatin** [comment]
hey @stratospher  ,  Ping for the video name and description..

**Comment by VirdiJatin** [comment]
Hey @stratospher , @rajarshimaitra reping for video elemnts..

**Comment by stratospher** [comment]
p2p: Fill reconciliation sets (Erlay)

This PR introduces tracking of per-peer reconciliation sets containing transactions to be exchanged efficiently using minisketch. Currently all the transaction are flooded to each peer via flooding algorithm . The section to be reviewed is txreconciliation.h/txreconciliation.cpp and   txreconciliation_tests.cpp.

**Comment by VirdiJatin** [comment]
@paperpsych ping for thumbnail..

**Comment by paperpsych** [comment]
@VirdiJatin 
![Bitshala RC (130624)](https://github.com/user-attachments/assets/d8f408fa-7d51-4e88-8cd1-4af712a48bd2)
